src/status_im/contexts/wallet/common/utils.cljs

```diff
@@ -36,6 +36,31 @@
        (map (comp :raw-balance val))
        (reduce money/add)))
 
+(defn extract-exponent
+  [s]
+  (if-let [index (string/index-of s "e")]
```

Comment:
nit:
"e" is related to euler's number? maybe we put it in a def
e.g `(def eulers-number "e")`

Reply:
@J-Son89 "e" is exponent in a number, i.e, `0.1234774e-7`. We are extracting the number after the exponent.